OneCommunityGlobal/HGNRest
# Vishala Consumables API

# Description
This PR is about the Consumables List View from Phase 2 with Sorting based on project name and item name , Filtering based on selected values of project and consumable and Modal implementation for viewing update and purchase history.
Implements # (WBS) 8.1.2

## Related PRS (if any):
To test this backend PR you need to checkout the [#1758](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1758) frontend PR.

## Main changes explained:
- Created a fetch api for consumables

## How to test:
1. check into current branch
2. do `npm run build` and `npm run start` to run this PR locally
3. Test api : http://localhost:4500/api/bm/consumables
